baseapp/baseapp_test.go

```diff
@@ -619,7 +623,9 @@ func TestInitChainer(t *testing.T) {
 	require.Equal(t, value, res.Value)
 }
 
-func TestInitChain_ProtocolVersionSetToZero(t *testing.T) {
+func TestInitChain_AppVersionSetToZero(t *testing.T) {
+	const expectedAppVersion = uint64(0)
```

Comment:
```suggestion
	const expectedAppVersion uint64
```

Reply:
Whoops, apparently constant needs to have a value as opposed to a var. Getting a syntax error. Constant makes more sense here so I changed it back